spec/intl.html

```diff
@@ -1401,11 +1401,66 @@ <h1>
           <dl class="header">
             <dt>description</dt>
             <dd>
-              It performs implementation-defined processing to convert _fields_ that describes a date in the calendar represented by _calendar_ to the ISO calendar and returns the corresponding day, month and year values as an ISO Date Record.
-              It may throw a *TypeError* exception if the own data properties present on _fields_ are insufficient to calculate an ISO date in _calendar_.
-              It may throw a *RangeError* exception if the values of the own data properties of _fields_ are out of range for _calendar_ and _overflow_ is *"reject"*, or if the date described by _fields_ forms a date outside the range allowed by ISODateTimeWithinLimits.
+              It performs implementation-defined processing to convert _fields_, that describes a date in the calendar represented by _calendar_, to the ISO calendar.
+              It performs the overflow correction given by _overflow_ on the date described by _fields_ in order to arrive at a valid month code and day pair in _calendar_.
+              For *"reject"*, values that do not form a valid date cause an exception to be thrown, as described below.
+              For *"constrain"*, values that do not form a valid date are clamped to the correct range.
```

Comment:
At this point PrepareTemporalFields has already validated the day and month with ToPositiveIntegerWithTruncation, so it's clarified elsewhere that too-low values throw a RangeError. I guess in light of your other comment below, if the first days of a month are skipped then a positive value could still be too low, in which case it should be clamped exactly as described here. So I don't know that we need to change this.

I was thinking of adding something like "clamped to the correct range, similar to [RegulateISODate](https://tc39.es/proposal-temporal/#sec-temporal-regulateisodate)" but I don't know if that would actually cause more confusion since this is not the ISO calendar.

Reply:
If <1 values are already handled elsewhere, then I don't think any change here is needed.